services/Donkey/src/donkey/core.clj

```diff
@@ -219,13 +216,21 @@
    :art-id "donkey"
    :service "donkey"})
 
+(def exception-handlers
+  {:handlers {::ex/request-validation (as-de-exception-handler ex/request-validation-handler ec/ERR_ILLEGAL_ARGUMENT)
+              :invalid-configuration (as-de-exception-handler invalid-cfg-handler ec/ERR_CONFIG_INVALID)
+              ::ex/response-validation (as-de-exception-handler ex/response-validation-handler ec/ERR_SCHEMA_VALIDATION)
+              ::ex/default unchecked-handler}})
+
```

Comment:
Being able to easily distinguish request validation errors from response validation errors will be *nice*!